test/jobs/skip-job-preview.ts

```diff
@@ -216,8 +220,32 @@ describe('Skipping job preview', function () {
 
     for (const [httpMethod, skipPreviewEndpointHook] of Object.entries(skipPreviewEndpointHooks)) {
       describe(`Skipping preview using ${httpMethod}`, function () {
+        describe('When an end user request results in a job in the previewing state', function () {
+          hookRangesetRequest('1.0.0', 'C1245618475-EEDTEST', 'all', { query: { maxResults: 500, format: 'application/x-zarr' }, username: 'joe' });
+          hookRedirect('joe');
 
-        describe('When a job is previewing', function () {
+          it('puts the job in the previewing state', function () {
+            const job = JSON.parse(this.res.text);
+            expect(job.status).to.eql('previewing');
+          });
+
+          describe('when skipping the preview it sets the job to the running status', async function () {
```

Comment:
I tried many different things, but the only way I could get access to the job ID to use to call skipPreview was from within a before block, so that's why the test setup describe/before/it looks so clunky.